### PR TITLE
Mepts 1066 | TX TB Differences

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java [hunk 1]
```diff
@@ -104,15 +104,15 @@ public static String inTBProgramWithinReportingPeriodAtLocation(Integer tbProgra
    * @return String
    */
   public static String pulmonaryTB(
-      Integer encounterTypeId, Integer pulmonaryTBConcept, Integer yesConcept) {
+      Integer encounterTypeId, Integer otherDiagnosisConcept ,Integer pulmonaryTBConcept) {
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java:116]: Can you please update the spec @TinoMuianga

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
Please update the spec

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java (hunk 1 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java:107]: Spec Updated

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**